resources/server_install.rb

```diff
@@ -25,6 +25,8 @@
 property :port,              [String, Integer], default: 5432
 property :initdb_locale,     String, default: 'UTF-8'
 
+include PostgresqlCookbook::Helpers
```

Comment:
Were these not included before? Why are they being added now?

Reply:
Ah. In that case I'd add it in to `action_class`.

Reply:
The action_class block is the provider portion of the file. So it only works for properties methods in the `action` block. Now I think of it, that's probably why it's called the action class.

Reply:
Oh right yeah you need that for a property, nevermind then.